### ISSUE TITLE
Links to Mac and Linux 2.0.12 builds are broken

### ISSUE THREAD

**Issue by diamondap** [APTrust/dart#491]
Links to download the Mac and Linux builds on https://aptrust.github.io/dart-docs/ both result in "No such key" errors from S3. Travis is building the dmg and deb files, but it's not pushing them correctly to S3.

**Comment by diamondap** [comment]
Travis was not creating the dmg or deb files because of a series of cascading failures related to Apple code signing and notarization. These are fixed now, and the 2.0.12 builds linked from https://aptrust.github.io/dart-docs/ are available.

travis config + apple code signing + notarization + node.js + node-gyp compilation failures + electron + electron-builder + electron-notarize = 🤯 

Complexity kills.

Due to changes in some underlying libraries, we should probably tag a new release.

- [ ] Tag v2.0.13

**Comment by diamondap** [comment]
There were two main issues causing the Apple build failure:

* Apple stopped recognizing DART's app-specific password for code signing and notarization. (No idea why.) We had to generate a new one.
* The latest version of ssh2-sftp-client includes C code that produces unsigned binaries that electron-builder can't seem to figure out how to sign. We reverted from v7.0.3 to v6.0.1 to solve this.

These issues are fixed in release 2.0.13. Both the Mac and Linux builds are now up on S3. See https://aptrust.github.io/dart-docs/ for links to the latest downloads.
